### PR TITLE
[type:fixbug]fix bug :after add paged is fail

### PR DESCRIPTION
**Pull request by li-keguo**
about:  https://github.com/apache/incubator-shenyu/issues/3062
 

fix ： Pagination fails after adding or editing

### CODE REVIEW

**Comment by dengliming** [comment]
@daiwenyu cc : )

File: src/routes/System/PluginHandle/index.js [hunk 1]
```diff
@@ -166,7 +166,7 @@ export default class PluginHandle extends Component {
                       type: "pluginHandle/fetch",
                       payload: {
                         pluginId: searchPluginId,
-                        currentPage: {currentPage},
+                        currentPage: 1,
```
Comment (daiwenyu) [inline src/routes/System/PluginHandle/index.js:169]: just use currentPage, remove {}

File: src/routes/System/PluginHandle/index.js [hunk 2]
```diff
@@ -228,7 +228,7 @@ export default class PluginHandle extends Component {
                   type: "pluginHandle/fetch",
                   payload: {
                     pluginId: searchPluginId,
-                    currentPage: {currentPage},
+                    currentPage: 1,
```
Comment (daiwenyu) [inline src/routes/System/PluginHandle/index.js:231]: same too, thanks

**Comment by daiwenyu** [comment]
@li-keguo Hi，buddy about src/routes/System/PluginHandle/index.js , I think the current page number should be kept，just remove {}, thanks

**Comment by li-keguo** [comment]
> @li-keguo Hi，buddy about src/routes/System/PluginHandle/index.js , I think the current page number should be kept，just remove {}, thanks

@daiwenyu Hi  it's ok to do this,
But I encountered a bug like this before：
There are query conditions before adding, but I didn't click the query button, my page number is on the last page，
After adding, the list cannot be queried for data。
I think all add needs to be reset after adding

**Comment by li-keguo** [comment]
e..... new condition reset too...

image

**Comment by li-keguo** [comment]
i try to fix it，

**Review by daiwenyu — APPROVED** [review]

**Pull request merged**